core/src/main/resources/jenkins/security/Messages_ru.properties

```diff
@@ -0,0 +1,56 @@
+# The MIT License
+#
+# Copyright 2022 CloudBees, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+ApiTokenProperty.DisplayName=Токен API
+ApiTokenProperty.ChangeToken.TokenIsHidden=Токен скрыт
+ApiTokenProperty.ChangeToken.Success=<div>Токен обновлён. Новый токен см. в поле сверху.</div>
+ApiTokenProperty.ChangeToken.SuccessHidden=<div>Токен обновлён. Чтобы увидеть токен, войдите в систему как этот \
```

Comment:
"войдите в систему как этот пользователь" sounds very unnatural. Not yet sure how to word it better. Also missing context.

Reply:
```// you are the user or you have ADMINISTER permission
            u.checkPermission(Jenkins.ADMINISTER);
...
...
return HttpResponses.html(p.hasPermissionToSeeToken()
                    ? Messages.ApiTokenProperty_ChangeToken_Success()
                    : Messages.ApiTokenProperty_ChangeToken_SuccessHidden());
```
Looks like it's either you yourself or an admin that can generate this token for you, but only you can see it.If an admin generated this token, they get this message instead of not seeing it.
```
<div>Updated. You need to login as the user to see the token</div>
```

Reply:
Then I'd consider "войдите используя учетную запись этого пользователя" or something close to that.